Polygon_mesh_processing/test/Polygon_mesh_processing/test_pmp_locate.cpp

```diff
@@ -475,11 +520,12 @@ void test_locate_in_face(const G& g,
     neigh_loc.second[(neigh_hd_id+2)%3] = FT(0);
 
     PMP::locate_in_adjacent_face(loc, neigh_f, g);
+    if (std::is_same<K, EPECK>()) {
+      assert(PMP::locate_in_common_face<FT>(loc, neigh_loc, g));
 
-    assert(PMP::locate_in_common_face<FT>(loc, neigh_loc, g));
-
-    assert(PMP::locate_in_common_face<FT>(loc, p, neigh_loc, g, CGAL::parameters::vertex_point_map(vpm).geom_traits(K())));
-    assert(PMP::locate_in_common_face<FT>(loc, p, neigh_loc, g, CGAL::parameters::vertex_point_map(vpm).geom_traits(K()), 1e-7));
+      assert(PMP::locate_in_common_face<FT>(loc, p, neigh_loc, g, CGAL::parameters::vertex_point_map(vpm).geom_traits(K())));
+      assert(PMP::locate_in_common_face<FT>(loc, p, neigh_loc, g, CGAL::parameters::vertex_point_map(vpm).geom_traits(K()), 1e-7));
+    }
```

Comment:
Everything above (since ~ L. 430) is construction-less and doesn't need to be EPECK only

Reply:
I tested it with EPICK and there was at least one exception. Due to the use of random numbers, it is not that easy to reproduce.
The assert in line 526 failed with the following seeds & models:
1667563258 data-coref/elephant_split_2.off
1667562794 & 1667563434 meshes/mech-holes-shark.off

Reply:
On Archlinux-clang-release

Reply:
@MaelRL do you consider this as resolved?